kiali.go

```diff
@@ -90,6 +90,37 @@ func main() {
 	status.Put(status.CoreCommitHash, commitHash)
 	status.Put(status.ContainerVersion, determineContainerVersion(version))
 
+	// check kiali version compatibility with istio
+	meshName, meshVersion, err := status.GetMeshInfo()
```

Comment:
This is just a question I have: I haven't looked but - does this status.GetMeshInfo require the user to be authenticated in order to get the MeshInfo data from the server? I assume this will work regardless of the auth strategy being used.

Reply:
In my view, querying version info from mesh service does not require to be authenticated.